bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java

```diff
@@ -150,7 +152,7 @@ public void processResult(int rc, String path, Object ctx) {
                     final AsyncCallback.VoidCallback stub = this;
                     scheduler.submit(new Runnable() {
                         @Override
-                        public final void run() {
+                        public void run() {
```

Comment:
A checkstyle rule requires to remove this modifier?

Reply:
yes, actually it does. Reverting that change results in:

    [ERROR] /Users/acoburn/Projects/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/AbstractHierarchicalLedgerManager.java:155:32: Redundant 'final' modifier. [RedundantModifier]